### PR TITLE
Feature HTML deployment under guides

### PR DESCRIPTION
**Pull request by rhodinemma**
# Description

This adds to the existing deployment guides [React, Flask, NodeJS, Laravel] in the user documentation.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## How Can This Been Tested?

Pull and run this branch locally to checkout the Deployment Guides section to review the HTML guide.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

# Screenshots

image

### CODE REVIEW

**Review by mosesmulumba — APPROVED** [review]
💪💪💪

File: cranecloud/docs/guides/deployHtmlApp.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+Getting an application running on Crane Cloud is essentially working out how to package it as a deployable image. Once packaged it can be deployed to the Crane Cloud platform.
+
+In this guide, we will learn how to containerize a sample HTML Application using Docker and then deploy to Crane Cloud.
+
+In this guide, we will learn how to containerize a sample HTML Application using Docker and then deploy to Crane Cloud. See [demo here](https://my-html-app-578060fa-3901-4464-8f3d.cranecloud.io/)
```
Comment (e-ian) [inline cranecloud/docs/guides/deployHtmlApp.md:5]: @rhodinemma repetition of the preceding line 3 above.

File: cranecloud/docs/guides/deployHtmlApp.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+2. Docker installed on your machine. If you haven't got Docker installed, you can use [this resource](https://docs.docker.com/get-docker/) to get you up and running.
+
+3. Docker Hub account. If you need to create one first, [click here.](https://hub.docker.com/signup)
+
+## Create a Dockerfile for your app
+
+In our application's directory, we shall create a `Dockerfile` and in it, let us write the following commands
+
+```js
```
Comment (e-ian) [inline cranecloud/docs/guides/deployHtmlApp.md:21]: `js` on line 21 here seems redundant and confusing to the reader

**Review by e-ian — CHANGES_REQUESTED** [review]
Well written. Just some small nits to consider

File: cranecloud/docs/guides/deployHtmlApp.md (hunk 1 above)
Comment (rhodinemma) [reply cranecloud/docs/guides/deployHtmlApp.md:5]: my bad! let me resolve that. thanks for noticing.

File: cranecloud/docs/guides/deployHtmlApp.md (hunk 2 above)
Comment (rhodinemma) [reply cranecloud/docs/guides/deployHtmlApp.md:21]: thank you.

Comment (rhodinemma) [reply cranecloud/docs/guides/deployHtmlApp.md:21]: I intentionally put the "js" to give the Dockerfile color while in the IDE, and has not effect on the UI docs.

**Review by e-ian — APPROVED** [review]
LGTM

**Review by LanternNassi — APPROVED** [review]

**Review by neelxie — APPROVED** [review]

**Pull request merged**